src/pages/Login/Login.tsx

```diff
@@ -24,7 +24,14 @@ const Login = () => {
 
   const handleToken = (token: string) => {
     localStorage.setItem('token', token);
-    setRole(getRole());
+    const role = getRole();
+    setRole(role);
+
+    if (role === 'USER') {
+      navigate('/');
```

Comment:
Can we have users role types somewhere in consts so we don't type it every time its error prone?